includes/Maps_MappingService.php

```diff
@@ -248,6 +248,17 @@ public function addResourceModules( $modules ) {
 	}
 
 	/**
+	 * Adds the layer dependencies.
+	 *
+	 * @param array $dependencies
+	 */
+	public function addLayerDependencies( array $dependencies ) {
```

Comment:
Why does this have Layer in its name? The code is not specific to layers at all. Or is it reasonable to expect it will be in the future?